src/hooks/useBlogs.js

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useState } from 'react';
+import { fetchBlogs } from '../services/blogs';
+
+export function useBlogs() {
+  const [blogs, setBlogs] = useState([]);
+
+  useEffect(() => {
+    async function fetchData() {
+      const data = await fetchBlogs();
+      setBlogs(data);
+    }
+    fetchData();
+  }, []);
+
+  return { blogs };
```

Comment:
Since you are only returning one thing here you don't need to stuff it inside an object. You can simply `return blogs`.